cnf-certification-test/accesscontrol/rbac/roles.go

```diff
@@ -36,6 +36,9 @@ type CrdResource struct {
 	ShortNames                      []string
 }
 
+// GetCrdResources retrieves a list of custom resource definitions (CRDs) and extracts relevant information into a list of CrdResource structs.
```

Comment:
GetCrdResources retrieves a list of custom resource definitions (CRDs) and extracts relevant information into a list of CrdResource structs. -> getCrdResources converts a list of apiextv1.CustomResourceDefinition structs into a list of list of CrdResource structs